server/pages/index.js

```diff
@@ -130,6 +137,9 @@ function getDefaultContext( request ) {
 		initialServerState = getInitialServerState( serializeCachedServerState );
 	}
 
+	const acceptedLanguages = getAcceptedLanguagesFromHeader( request.headers[ 'accept-language' ] );
+	const pride = indexOf( prideLanguages, '*' ) > -1 || intersection( prideLanguages, acceptedLanguages );
```

Comment:
`Array.prototype.indexOf` is available on all modern browser I believe

Reply:
Also you probably meant `intersection( prideLanguages, acceptedLanguages ).length > 0`